functional-tests/views/index.pug

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+html
+  head
+    <meta charset="UTF-8">
+    link(rel="stylesheet" href='annotations.css')
+    script(src='annotations.js')
+    link(rel="stylesheet" href='https://cdn01.boxcdn.net/platform/preview/1.37.0/en-US/preview.css')
```

Comment:
How are you planning on keeping this up to date?

Reply:
Since Preview shouldn't affect annotations anymore, using an older version of preview shouldn't make a difference. The only reason I could see to update this regularly would be if there's drastic changes to how Preview and Annotations interact.

Reply:
Maybe you can make this an env variable just incase?

Reply:
also will have to update this after your css class gets added into preview.

Reply:
the CSS change is in the annotations codebase so it shouldn't affect the preview version

Reply:
ah right